### PR TITLE
Fix more integ test and increase waitFor 

### PR DESCRIPTION
**Pull request by jinsoor-amzn**
*Description of changes:*
Increase WaitFor
Fix more integtest 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/ActionRetryIT.kt [hunk 1]
```diff
@@ -57,16 +58,50 @@ class ActionRetryIT : IndexStateManagementRestTestCase() {
 
         // Third execution is to fail the step second time.
         updateManagedIndexConfigStartTime(managedIndexConfig, Instant.now().minusSeconds(58).toEpochMilli())
-        Thread.sleep(3000)
+        val responseFirst: Response = waitFor {
```
Comment (qreshi) [inline src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/ActionRetryIT.kt]: Can replace this with Drew's ```getExplainMap()```. Same goes for lines ```98``` and ```175``` of this file.

File: src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/IndexStateManagementHistoryIT.kt [hunk 2]
```diff
@@ -45,10 +45,12 @@ class IndexStateManagementHistoryIT : IndexStateManagementRestTestCase() {
         createIndex(indexName, policyID)
         resetHistorySetting()
 
-        Thread.sleep(2000)
+        val managedIndexConfig: ManagedIndexConfig? = waitFor {
```
Comment (qreshi) [inline src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/IndexStateManagementHistoryIT.kt]: Can replace with ```getExistingManagedIndexConfig()```. Same goes for lines ```118, 188, 258 and 360```

File: src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/ActionRetryIT.kt [hunk 3]
```diff
@@ -42,89 +36,52 @@ class ActionRetryIT : IndexStateManagementRestTestCase() {
...
-        }
+        waitFor { assertEquals(policyID, getExplainManagedIndexMetaData(indexName).policyID) }
 
         // Second execution is to fail the step once.
         updateManagedIndexConfigStartTime(managedIndexConfig, Instant.now().minusSeconds(58).toEpochMilli())
-        Thread.sleep(3000)
+
+        waitFor {
+            val managedIndexMetaData = getExplainManagedIndexMetaData(indexName)
+            logger.info("print managedIndexMetaData $managedIndexMetaData")
```
Comment (qreshi) [inline src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/ActionRetryIT.kt]: Was this used for debugging? May not provide useful information to someone unless they knew where it was coming from.

Comment (jinsoor-amzn) [reply src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/ActionRetryIT.kt]: oops.. Removed the logging.

**Pull request merged**